src/qt/peertablemodel.h

```diff
@@ -74,22 +62,25 @@ class PeerTableModel : public QAbstractTableModel
 
     /** @name Methods overridden from QAbstractTableModel
         @{*/
-    int rowCount(const QModelIndex &parent) const override;
-    int columnCount(const QModelIndex &parent) const override;
-    QVariant data(const QModelIndex &index, int role) const override;
-    QVariant headerData(int section, Qt::Orientation orientation, int role) const override;
-    QModelIndex index(int row, int column, const QModelIndex &parent) const override;
+    int rowCount(const QModelIndex& parent = QModelIndex()) const override;
```

Comment:
nit: if one likes C++11 toys, one could write `= {}` instead of ` = QModelIndex()`.

Reply:
As stated in the commit description, the goal is to follow Qt [docs](https://doc.qt.io/qt-5/qabstracttablemodel.html#index).